react/features/base/avatar/components/native/Avatar.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+
+import React from 'react';
+import { Image, Text, View } from 'react-native';
+
+import { connect } from '../../../redux';
+import { type StyleType } from '../../../styles';
+
+import AbstractAvatar, {
+    _mapStateToProps,
+    type Props as AbstractProps,
+    DEFAULT_SIZE
+} from '../AbstractAvatar';
+
+import RemoteAvatar from './RemoteAvatar';
+import styles from './styles';
+
+type Props = AbstractProps & {
+
+    /**
+     * External style of the component.
+     */
+    style?: StyleType
+}
+
+export const DEFAULT_AVATAR = require('../../../../../../images/avatar.png');
+
+/**
+ * Implements an avatar component that has 4 ways to render an avatar:
+ *
+ * - Based on an explicit avatar URI, if provided
```

Comment:
when are we using this first one, instead of using a participant?

Reply:
E.g. in case of the IncomingCallPage